templates/ethstore.html

```diff
@@ -0,0 +1,524 @@
+                  <td>SE + 225</td>
+                </tr>
+                <tr>
+                  <td>Active validators (VS):</td>
+                  <td>Validators that have been active from start SE to end EE [SE,EE)</td>
+                </tr>
+                <tr>
+                  <td>Sum rewards (RS):</td>
+                  <td>Sum of all rewards (consensus, tx) in VS that have been generated in epochs [SE,EE)</td>
+                </tr>
+                <tr>
+                  <td>Sum effective balances (EB):</td>
+                  <td>Sum of all effective balances at epoch SE of all validators in VS</td>
+                </tr>
+                <tr>
+                  <td>ETH.STORE:</td>
+                  <td>(RS/EB)*365</td>
+                </tr>
+              </tbody>
+            </table>
+          </div>
+
+          <br />
+          <p>
+            Calculations for <strong><span aria-ethereum-date="{{ .Data.YesterdayTs }}" aria-ethereum-date-format="dd MMMM yyyy"></span> 12:00:23 UTC - <span aria-ethereum-date="{{ addI64 .Data.YesterdayTs 86400 }}" aria-ethereum-date-format="dd MMMM yyyy"></span> 12:00:22 UTC:</strong>
+          </p>
+
+          <div class="table-responsive">
+            <table class="table">
+              <thead>
+                <tr>
+                  <th>Variable</th>
+                  <th>Description</th>
+                </tr>
+              </thead>
+              <tbody>
+                <tr>
+                  <td>Start epoch (SE):</td>
+                  <td>Epoch {{ .Data.StartEpoch }}</td>
+                </tr>
+                <tr>
+                  <td>End epoch (EE):</td>
+                  <td>Epoch {{ addUI64 .Data.StartEpoch 225 }}</td>
+                </tr>
+                <tr>
+                  <td>Active validators (VS):</td>
+                  <td>in timeframe [{{ .Data.StartEpoch }},{{ addUI64 .Data.StartEpoch 225 }})</td>
+                </tr>
+                <tr>
+                  <td>Sum rewards (RS):</td>
+                  <td>{{ .Data.YesterdayRewards }} Ether</td>
+                </tr>
+                <tr>
+                  <td>Sum effective balances (EB):</td>
+                  <td>{{ .Data.YesterdayEffectiveBalance }} Ether</td>
+                </tr>
+                <tr>
+                  <td>ETH.STORE:</td>
+                  <td>{{ .Data.ProjectedAPR }} % = ({{ .Data.YesterdayRewards }} / ({{ .Data.YesterdayEffectiveBalance }})) * 365</td>
+                </tr>
+              </tbody>
+            </table>
+          </div>
+        </div>
+
+        <button type="button" class="collapsible">Is the ETH.STORE code Open-Source?</button>
+        <div class="content">
+          <p>Yes, you can run the calculations yourself with our open source ETH.STORE tool and verify the integrity of the ETH.STORE listed here. Check out <a href="https://github.com/gobitfly/eth.store/" target="_blank">ETH.STORE on Github</a>.</p>
```

Comment:
The link to github in https://staking.ethermine.org/statistics is orange while it is blue here. Should this be harmonized?

Reply:
No, won't change